content/03.methods.md

```diff
@@ -117,6 +131,18 @@ We annotated putative driver fusions and prioritized fusions lists with kinases,
 We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
 
 ### Clinical Data Harmonization
+#### WHO Classification of Disease Types
```

Comment:
```suggestion

#### WHO Classification of Disease Types
```

Reply:
@jaclyn-taroni this sentence describes the column for WHO classification: 

> The `broad_histology` denotes the broad 2016 WHO classification [doi:10.1007/s00401-016-1545-1] for each tumor.

Reply:
This was a suggestion about spacing when reading the Markdown source.

Reply:
got it, sorry I could not respond directly to your question above re: WHO column definition, so that was in response to that comment

Reply:
Ah, I think I know what comment you are referencing. I meant can you state what column contains the medulloblastoma subtype info.

Reply:
ahh gotcha - just committed.